cli/ssh.go

```diff
@@ -143,13 +143,11 @@ func (r *RootCmd) ssh() *clibase.Cmd {
 			case "no":
 				wait = false
 			case "auto":
-				switch workspaceAgent.StartupScriptBehavior {
-				case codersdk.WorkspaceAgentStartupScriptBehaviorBlocking:
-					wait = true
-				case codersdk.WorkspaceAgentStartupScriptBehaviorNonBlocking:
-					wait = false
-				default:
-					return xerrors.Errorf("unknown startup script behavior %q", workspaceAgent.StartupScriptBehavior)
+				for _, script := range workspaceAgent.Scripts {
```

Comment:
Seeing this change, I believe old clients connecting to coderd will be broken by this since the API no longer includes the startup script behavior.

Reply:
I'll fix this!

Reply:
@mafredri do you think I should just include the field and leave it as a static value to not break old clients?

Reply:
@kylecarbs I'm assuming static would be non-blocking? For anyone using `blocking` I suppose it would still be a breaking change. But I'm not sure it's worth including so much legacy to keep piping it to the DB, so I think it's acceptable. But if we leave it like that, I think we should mention it in the release notes.